dietpi/dietpi-software

```diff
@@ -11929,40 +11931,45 @@ _EOF_
 
 		fi
 
-		software_id=183 # Bitwarden_RS
+		software_id=183 # Vaultwarden
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
 
 			Banner_Configuration
 
 			# User
-			Create_User -d /mnt/dietpi_userdata/bitwarden_rs bitwarden_rs
+			Create_User -d /mnt/dietpi_userdata/vaultwarden vaultwarden
 
 			# Data and config dir
-			G_EXEC mkdir -p /mnt/dietpi_userdata/bitwarden_rs
+			G_EXEC mkdir -p /mnt/dietpi_userdata/vaultwarden
+
+			# Config: Rename old Bitwarden_RS due to project name change
+			if [[ -d '/mnt/dietpi_userdata/bitwarden_rs' ]]; then
+				G_EXEC mv /mnt/dietpi_userdata/bitwarden_rs /mnt/dietpi_userdata/vaultwarden
```

Comment:
This needs to be done before the directory is created above, else it lands inside as sub directory. Or if-then-else?
```
if [[ -d '/mnt/dietpi_userdata/bitwarden_rs' ]]
then
	G_EXEC mv /mnt/dietpi_userdata/{bitwarden_rs,vaultwarden}

elif [[ ! -d '/mnt/dietpi_userdata/vaultwarden' ]]
then
	G_EXEC mkdir -p /mnt/dietpi_userdata/vaultwarden
fi
```

Reply:
ok I add the section but I guess we would need to agree if we force a reinstall on next `dietpi-update` or not as this will have impact an what to place on reinstall

Reply:
I tried to rename `bitwarden_rs.env` but not sure if I used correct syntax 🙄